app/controllers/drugs_controller.rb

```diff
@@ -29,7 +29,12 @@ def name_suggestion
     if params[:q].blank?
       render json:  {errors: ['Must specify a query with parameter q']}, status: :bad_request
     else
-      render json: DrugNameSuggestion.suggestions_for_name(params[:q]), status: :ok
```

Comment:
Is `DrugNameSuggestion` used anywhere else with this gone? If not, we should delete it.